README.md

```diff
@@ -20,3 +22,23 @@ Example using the [example_add_i32](https://github.com/stellar/rs-stellar-contra
 $ stellar-contract-cli invoke --file example_add_i32.wasm --fn add --arg i32:1 --arg i32:2
 i32:3
 ```
+
+### Calling a host function
+
+For both options below, the file should contain a JSON representation of the `SCVec` xdr object, which is passed directly to `invoke_function` in the host. The order of the items in the `SCVec` is specified below.
+
+#### Creating a contract
+
+```
+// args = SCVec(contractCode, salt, ed25519, signature)
+stellar-contract-cli invoke --file <JSONARGS> create-contract
```

Comment:
I think we should replace `invoke create-contract` with a `deploy`. We plan to add functionality for deploying to a running network (#41), and this function is identical to that except that we're not submitting a tx and it is all happening locally. As part of #41 we can figure out how to distinguish them.

Reply:
Done. `deploy` just commits a contractID and WASM blob to storage. This can easily be extended to  create a contract using the host function in the future.